README.md

```diff
@@ -27,10 +27,9 @@ The following matrix shows which Kubernetes versions and platforms are supported
 |     `0.3`    |   `1.8`-`1.9`       | Yes | No  |
 |     `1.0`    |   `1.9`-`1.10`      | Yes | Yes |
 
-## Quickstart
+## Installation
 
-* [AKS Quickstart](docs/quickstart-aks.md)
-* [GKE Quickstart](docs/quickstart-gke.md)
+Please follow the [installation guide](docs/install.md) to install BKPR to your cluster.
```

Comment:
I was conflicted about "install to", it sounded wrong, so I checked other docs online and the general convention seems to be "install on a cluster" and in some cases "install in a cluster environment" refs. https://docs.oracle.com/cd/E19787-01/820-2555/z40001fb1003552/index.html and https://documentation.commvault.com/commvault/v11_sp6/article?p=deployment/install/t_install_server_pkg_cluster.htm

I think we should say "install BKPR on your cluster"